cli.js

```diff
@@ -53,13 +59,43 @@ function isFile(path) {
 	}
 }
 
+var DEFAULT_PLUGINS = ['gifsicle', 'jpegtran', 'optipng', 'svgo'];
+
 function run(src, dest) {
-	var imagemin = new Imagemin()
-		.src(src)
-		.use(Imagemin.gifsicle(cli.flags))
-		.use(Imagemin.jpegtran(cli.flags))
-		.use(Imagemin.optipng(cli.flags))
-		.use(Imagemin.svgo());
+	var plugins;
+	if (Array.isArray(cli.flags.plugin)) {
+		plugins = cli.flags.plugin;
+	} else if (cli.flags.plugin) {
+		plugins = [cli.flags.plugin];
+	} else {
+		plugins = DEFAULT_PLUGINS;
+	}
+
+	var imagemin = new Imagemin().src(src);
+
+	plugins.forEach(function (name) {
+		var plugin;
+		switch (name) {
+			case 'gifsicle':
+			case 'jpegtran':
+			case 'optipng':
+				plugin = Imagemin[name](cli.flags);
+				break;
+			case 'svgo':
+				plugin = Imagemin.svgo();
+				break;
+			default:
+				try {
+					plugin = require('imagemin-' + name)(cli.flags);
+				} catch (e) {
+					console.error("unknown plugin", name);
```

Comment:
We should probably also include the following message as a lot of users seem to consistently do this mistake:

> Maybe you forgot to install the plugin. You can install it with: `$ npm install imagemin-name`.